7-image-and-video/1-comfyui/README.MD

```diff
@@ -0,0 +1,124 @@
+# ComfyUI API Wrapper
+
+This project provides a FastAPI-based wrapper around ComfyUI, making it easy to run Stable Diffusion workflows programmatically through a RESTful API.
+
+## Overview
+
+The ComfyUI API Wrapper starts a ComfyUI server in the background and exposes endpoints to:
+- Run Stable Diffusion workflows with custom parameters
+- Check server health status
+- Dynamically load and configure models
+
+The API automatically handles model downloading, workflow template substitution, and converting the generated images to base64 for easy integration with other services.
+
+## Installation
+
+1. Clone this repository:
+```bash
+git clone https://github.com/yourusername/comfyui-api-wrapper.git
+cd comfyui-api-wrapper
+```
+
+2. Install dependencies:
+```bash
+pip install -r requirements.txt
+```
+
+3. Clone ComfyUI repository (if not already included):
+```bash
+git clone https://github.com/comfyanonymous/ComfyUI
+pip install -r ComfyUI/requirements.txt
+```
+
+## Usage
+
+### Starting the Server
```

Comment:
mm surely this isn't needed since user isn't going to run locally? Guess they can if they have a gpu